cmd/atc/handler.go

```diff
@@ -165,7 +167,11 @@ func Handler(client *k8s.Client, cache *wasm.ModuleCache, controllers *atc.Contr
 
 			addRequestAttrs(r.Context(), slog.Bool("skipped", true))
 
-			json.NewEncoder(w).Encode(&review)
+			if err := json.NewEncoder(w).Encode(&review); err != nil {
```

Comment:
I don't believe we should be chaning the http response if we fail to write to it. The headers could be already sent, or a partial response made before the connection is broken.

Much better to simply log the error.